### PR TITLE
Add configuration value: compute.ops_on_diff_frames

### PR DESCRIPTION
**Pull request by itholic**
related with #724  i added compute.ops_on_diff_frames

### CODE REVIEW

File: databricks/koalas/config.py [hunk 1]
```diff
@@ -34,6 +34,12 @@
     # For example, this value determines whether the repr() for a dataframe prints out fully or
     # just a truncated repr.
     "display.max_rows": 1000,  # TODO: None should support unlimited.
+
+    # This determines whether or not to operate between two different dataframs.
+    # For example, 'combine_frames' function internally performs a join operation which can be
+    # expensive in general.
+    # So, if `compute.ops_on_diff_frames` variable is not True, that method throws an exception.
+    "compute.ops_on_diff_frames": True,
```
Comment (HyukjinKwon) [inline databricks/koalas/config.py]: It defaults `false`.

Comment (itholic) [reply databricks/koalas/config.py]: @HyukjinKwon Thanks. i've fix it.

File: databricks/koalas/utils.py [hunk 2]
```diff
@@ -39,11 +38,12 @@ def combine_frames(this, *args, how="full"):
     the columns names from both DataFrames
 
     It internally performs a join operation which can be expensive in general.
-    So, if `OPS_ON_DIFF_FRAMES` environment variable is not set,
+    So, if `compute.ops_on_diff_frames` variable is not True,
```
Comment (HyukjinKwon) [inline databricks/koalas/utils.py]: `variable` -> `option`.

Comment (HyukjinKwon) [reply databricks/koalas/utils.py]: `is not True` -> `is False`

**Pull request merged**